Lombiq.Vsix.Orchard.sln

```diff
@@ -1,26 +1,49 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29905.134
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Lombiq.Vsix.Orchard", "Lombiq.Vsix.Orchard\Lombiq.Vsix.Orchard.csproj", "{EF02401B-9EB3-4E06-8C34-8411700E4B58}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Lombiq.Vsix.Orchard16", "Lombiq.Vsix.Orchard\Lombiq.Vsix.Orchard16.csproj", "{EF02401B-9EB3-4E06-8C34-8411700E4B58}"

```

Comment:
Mivel a project át lenne nevezve, ezért szerintem a parent mappájának is követnie kéne.